internal/plugin/portable/factory.go

```diff
@@ -30,6 +30,17 @@ func (m *Manager) Source(name string) (api.Source, error) {
 	return runtime.NewPortableSource(name, meta), nil
 }
 
+func (m *Manager) GetSourcePlugin(name string) (plugin.EXTENSION_TYPE, string, string) {
```

Comment:
Is GetPluginMeta the same as this?

Reply:
GetPluginMeta do not have the install infomation of the plugin
 GetSourcePlugin need the install infomation of the plugin